.github/workflows/continous-integration.yml

```diff
@@ -101,11 +101,14 @@ jobs:
       # trying to run the `rasa` command once before the docs build.
       run: poetry run rasa --help
 
-    - name: Test Docs 📃
+    - name: Build Docs 📃
       run: |
         poetry run python -c "from scripts import release; release.generate_changelog('major.minor.patch')"
```

Comment:
wdyt about adding this directly to `make docs` so its easier to `make docs test-docs` locally?

Reply:
no we can't do that, it's used to generate the docs in the deployment pipeline

Reply:
but we could definitely modify test-docs to do that automatically

Reply:
ah makes sense 👍

Reply:
wait, doesn't the changelog still need to be generated though 🤔

Reply:
yes but it's not done like this. Look at the "major.minor.patch" argument :)

Reply:
I updated the workflow here: c55905a (and added a helper). It should be easier to use locally now 👍